Player/Player.php

```diff
@@ -40,18 +40,16 @@ class Player
      */
     public function __construct($client)
     {
-        if ($client instanceof GuzzleClient) {
+        if (is_array($client)) {
+            $this->clients = $client;
+        } else {
             $this->clients[] = $client;
-        } elseif (is_array($client)) {
-            foreach ($client as $c) {
-                if (!$c instanceof GuzzleClient) {
-                    throw new LogicException('Blackfire Player accepts a Guzzle client or an array of Guzzle clients.');
-                }
-
-                $this->clients[] = $c;
+        }
+    
+        foreach ($this->clients as $c) {
```

Comment:
this is a bad idea, as you start by assigning invalid data and then validate it. It is better to validate the data before altering the object with it.